src/Volume.elm

```diff
@@ -0,0 +1,213 @@
+module Volume exposing
+    ( Volume, CubicMeters
+    , cubicMeters, inCubicMeters
+    , milliliters, inMilliliters, liters, inLiters
+    , cubicInches, inCubicInches, cubicFeet, inCubicFeet, cubicYards, inCubicYards
+    , usLiquidGallons, inUsLiquidGallons, usDryGallons, inUsDryGallons, imperialGallons, inImperialGallons
+    , usLiquidQuarts, inUsLiquidQuarts, usDryQuarts, inUsDryQuarts, imperialQuarts, inImperialQuarts
+    --, usLiquidPints, usDryPints, imperialPints
+    --, usFluidOunces, imperialFluidOunces
+    )
+
+{-| A `Volume` represents a volume in cubic meters, cubic feet, liters,
+US liquid gallons, imperial fluid ounces etc. It is stored as a number of cubic meters.
+
+@docs Volume, CubicMeters
+
+
+## Metric
+
+@docs cubicMeters, inCubicMeters
+@docs milliliters, inMilliliters, liters, inLiters
+
+
+## Imperial
+
+@docs cubicInches, inCubicInches, cubicFeet, inCubicFeet, cubicYards, inCubicYards
+@docs usLiquidGallons, inUsLiquidGallons, usDryGallons, inUsDryGallons, imperialGallons, inImperialGallons
+@docs usLiquidQuarts, inUsLiquidQuarts, usDryQuarts, inUsDryQuarts, imperialQuarts, inImperialQuarts
+--@docs usLiquidPints, usDryPints, imperialPints
+--@docs usFluidOunces, imperialFluidOunces
```

Comment:
elm-format adds the extra `@docs` leader every time - have left these here for ref but will be sorted before it's ready to merge.